src/components/views/elements/AddressSelector.js

```diff
@@ -16,16 +16,19 @@ limitations under the License.
 
 'use strict';
 
-var React = require("react");
-var sdk = require("../../../index");
-var classNames = require('classnames');
+const React = require("react");
```

Comment:
if you're going to rewrite these (which I applaud), can you switch to `import` instead of `require`?